src/remote-config/condition-evaluator-internal.ts

```diff
@@ -173,18 +173,8 @@ export class ConditionEvaluator {
   }
 
   static hashSeededRandomizationId(seededRandomizationId: string): bigint {
-    // For consistency with the Remote Config fetch endpoint's percent condition behavior
-    // we use Farmhash's fingerprint64 algorithm and interpret the resulting unsigned value
-    // as a signed value.
-    let hash64 = BigInt.asIntN(64, farmhash.fingerprint64(seededRandomizationId));
-
-    // Manually negate the hash if its value is less than 0, since Math.abs doesn't
-    // support BigInt.
-    if (hash64 < 0) {
-      hash64 = -hash64;
-    }
-
-    return hash64;
+    const hex = createHash('sha256').update(seededRandomizationId).digest('hex');
```

Comment:
Thinking (no action req'd): we had a 64 bits hash before, so we don't need 256 bits, but sha256 is broadly available and there's no noticeable performance difference, so sha256 is fine.